app/views/components/_blue-box-campaign.html.erb

```diff
@@ -0,0 +1,6 @@
+<div class="app-c-native-blue-box-campaign__border">
+  <a class="app-c-native-blue-box-campaign__link" href="<%= href %>">
+    <span class="app-c-native-blue-box-campaign__title"><strong><%= title %></strong></span>
```

Comment:
I'm not sure this is the right place to use `<strong>` tags, a heading element may be more appropriate.